blocks/countdown/src/countdown-title.tsx

```diff
@@ -16,13 +16,33 @@ export const CountdownTitle: FunctionComponent<CountdownTitleProps> = ({
   const textareaEl = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
-    if (!textareaEl.current) return;
-    textareaEl.current.textContent = value ?? "";
-  }, [value]);
+    const textarea = textareaEl.current;
+    if (!textarea) return;
+    textarea.textContent = value ?? "";
+
+    // Calling this whenever value changes makes it so the cursor position is lost in some browsers
+    // (firefox and safari for example), so we leave it out of the dependency array
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
This means that if `value` is changed from the outside, this element will not update. 

One quick option to prevent the cursor issue is to only update the value if it is different from the element's current content.

Although to be honest I'm unsure why we're using a `contentEditable` div rather than an actual `input` here, which would probably avoid some of these issues.

Reply:
There's some differences from `input` and `textarea` elements, namely that they automatically expand to take a set width and don't really expand naturally as the value is changed.

Reply:
Made it so the value reacts to external changes in bfcf717bea86ee97f89575593564186883cbc974